Modules/IO/GE/src/itkGE4ImageIO.cxx

```diff
@@ -369,9 +370,7 @@ GE4ImageIO ::MvtSunf(int numb)
     sun_exp = 255;
   }
   dg_mantissa = dg_mantissa << 1;
-  int   sun_num = dg_sign | (sun_exp << smantlen) | ((dg_mantissa >> 9) & smantissa);
-  float x;
-  memcpy(&x, &sun_num, sizeof(x));
-  return x;
+  const unsigned int sun_num{ dg_sign | (sun_exp << smantlen) | ((dg_mantissa >> 9) & smantissa) };
```

Comment:
We could use multiplication and addition rather than shifts and bitwise-or, as in #3398.  (For those who didn't read #3398, ... that's a joke.  I prefer the latter.)